README.md

```diff
@@ -299,14 +299,41 @@ getBars(
 ) => Promise<BarsObject>
 ```
 
+#### Get Aggregates
+
+```ts
+getAggregates(
+  symbol: string,
+  timespan: 'minute', 'hour', 'day', 'week', 'month', 'quarter', 'year',
+  from: Date,
+  to: Date,
+) => Promise<AggregatesObject>
+```
+
+#### Last trade
+
+```ts
+lastTrade(
+  symbol: string)
+) => Promise<LastTradeObject>
+```
+
+#### Last quote
+
+```ts
+lastQuote(
+  symbol: string)
+) => Promise<LastQuoteObject>
+```
+
 ### Websockets
 
 * `let websocket = alpaca.websocket`: Create a websocket client instance.
 * `websocket.connect()`: Connect to the Alpaca server using websocket.
 * `websocket.subscribe(channels)`: Subscribe to the Alpaca server and possibly the Polygon server.
-    Possible channels: `['trade_updates', 'account_updates', 'T.*', 'Q.*', 'A.*', 'AM.*']`.
+    Possible channels: `['trade_updates', 'account_updates', 'T.*', 'Q.*', 'A.*', 'AM.*', 'polyfeed/T.*', 'polyfeed/Q.*', 'polyfeed/A.*', 'polyfeed'/AM.*]`.
```

Comment:
* is not allowed in polyfeed. I understand the intent here, but users might get confused.

Reply:
how about
`['trade_updates', 'account_updates', 'T.*', 'Q.*', 'A.*', 'AM.*', 'polyfeed/T.<symbol>', 'polyfeed/Q.<symbol>', 'polyfeed/A.<symbol>', 'polyfeed'/AM.<symbol>]` ?